src/services/s3Submit.ts

```diff
@@ -92,6 +92,7 @@ const getS3Instance = ({ credentials, s3: s3Meta }: S3Configuration) => {
     accessKeyId: credentials.AccessKeyId,
     secretAccessKey: credentials.SecretAccessKey,
     sessionToken: credentials.SessionToken,
+    correctClockSkew: true,
```

Comment:
Also an instance being created in downloadPreFillData()

Reply:
👍